bokeh/models/tiles.py

```diff
@@ -58,6 +58,10 @@ class MercatorTileSource(TileSource):
     resolution (plot_units / pixels) of minimum zoom level of tileset projection. None to auto-compute.
     """)
 
+    wrap_around_180 = Bool(default=True, help="""
```

Comment:
Maybe just `wrap_around`? With the scale mentioned in the help? I'm not aware of any other property names with digits in them. Obviously it works but it's a little incongruous to me.